src/openrct2-ui/windows/Options.cpp

```diff
@@ -1022,11 +1023,13 @@ static void window_options_mousedown(rct_window *w, rct_widgetindex widgetIndex,
             gDropdownItemsFormat[0] = STR_DROPDOWN_MENU_LABEL;
             gDropdownItemsFormat[1] = STR_DROPDOWN_MENU_LABEL;
             gDropdownItemsFormat[2] = STR_DROPDOWN_MENU_LABEL;
+            gDropdownItemsFormat[3] = STR_DROPDOWN_MENU_LABEL;
             gDropdownItemsArgs[0] = STR_SCALING_QUALITY_NN;
             gDropdownItemsArgs[1] = STR_SCALING_QUALITY_LINEAR;
             gDropdownItemsArgs[2] = STR_SCALING_QUALITY_ANISOTROPIC;
```

Comment:
We should really get rid of anisotropic. It only makes sense for slanted surfaces and in hindsight, adding it was a mistake. The question is should your new "smoothed" option take its place or be introduced as new one? Maybe changing the config entry to a textual one and reset everyone to NN, if they had it on something else, it would prompt them to go into options and discover this new one. Thoughts?

Likewise, maybe pushing on having a separate option to have NN on integer scales was not the smartest thing too. I initially imagined people would change scale more frequently, but I doubt that's the case, so we can just default to that.